consensus/src/block_storage/sync_manager.rs

```diff
@@ -83,9 +83,6 @@ impl BlockStore {
         )
         .await?;
 
-        self.insert_quorum_cert(sync_info.highest_commit_cert(), &mut retriever)
```

Comment:
we shouldn't mix the commit cert with ordering cert, this is added in sync_to_highest_commit_cert